<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputDialog</class>
 <widget class="QDialog" name="InputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>169</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="num1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="num2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="num3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="num5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="num6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="num4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="num8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="num9">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="num7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
